cloud/gcp/main.tf

```diff
@@ -19,20 +19,42 @@ provider "google" {
   request_timeout = "60s"
 }
 
+module "IAM" {
```

Comment:
the pattern we're following is that main is inside templates folder, so this would be cloud/gcp/templates/gcp/main.tf

Reply:
Why the gcp subdirectory after `templates` ? We are assuming this to be entirely GCP centric

Reply:
good point. we currently have https://github.com/seattle-uat/civiform/tree/main/cloud/azure/templates/azure_saml_ses and https://github.com/seattle-uat/civiform/tree/main/cloud/aws/templates/aws_oidc. So you can call this gcp_oidc? or maybe just "default"? The idea is that we might have slightly different variations of configs for the same provider.